files/zh-tw/learn/html/multimedia_and_embedding/images_in_html/index.md

```diff
@@ -73,7 +73,7 @@ slug: Learn/HTML/Multimedia_and_embedding/Images_in_HTML
 
 ![A basic image of a dinosaur, embedded in a browser, with "Images in HTML" written above it](basic-image.png)
 
-> **備註：** \<img>和\<video>之類的元素有時也稱為替換元素。 這是因為元素的內容和圖片大小是由外部（例如圖片或影音檔）所定義的，而不是由元素的內容定義。
+> [!NOTE] > \<img>和\<video>之類的元素有時也稱為替換元素。 這是因為元素的內容和圖片大小是由外部（例如圖片或影音檔）所定義的，而不是由元素的內容定義。
```

Comment:
Not correct here

Reply:
Even backslashes seem to trigger the Prettier bug, that's annoying...

Would you be down to edit this PR to make the appropriate changes?  My working tree is not empty at the moment!

Reply:
Resolved in 271adbf34c3f5173cb26da1fdde4282d8d7c801b